### PR TITLE
Refactor current selection into service

### PR DESCRIPTION
**Pull request by owennw**
Much better!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
-﻿﻿﻿<!DOCTYPE HTML>
+﻿﻿<!DOCTYPE HTML>
```
Comment (bjedrzejewski) [inline src/index.html]: Are you inserting the white space again here?

File: src/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OpenFin D3FC</title>
-    <link href="assets/css/style.css" rel="stylesheet" >
+    <link href="assets/css/style.css" rel="stylesheet">
     <link href="assets/css/d3fc.min.css" rel="stylesheet" />
     <link href="assets/css/jquery.mCustomScrollbar.css" rel="stylesheet" />
     <style>
```
Comment (bjedrzejewski) [inline src/index.html]: Since you are changing the way this <link> closes, do you want to remove other self closing links? This is the html5 practice.

**Comment by bjedrzejewski** [comment]
Great change, now the code is easier to understand.

**Comment by bjedrzejewski** [comment]
@owennw Maybe favourites can be refactored in similar way rather than using the events mechanic? Not necessarily now, but when we have more time?

File: src/index.html (hunk 1 above)
Comment (owennw) [inline src/index.html]: I have no idea -- using my diff tool I don't see a difference.. and if the illegal character were there it wouldn't pass linting..

Comment (bjedrzejewski) [inline src/index.html]: HTML doesn't get linted. Anyway, I was just curious if you can see something suspicious.

**Pull request merged**